bzt/modules/jmeter.py

```diff
@@ -2025,31 +2024,13 @@ def install(self):
         os.remove(jmeter_dist.name)
 
         if self.check_if_installed():
-            self.remove_old_jar_versions(os.path.join(dest, 'lib'))
+            cleaner = JarCleaner(self.log)
```

Comment:
Not only plugins:
DEBUG Engine.JMeterExecutor.JMeter.JarCleaner] Old jar removed commons-jexl-1.1.jar